feature/gribi/ate_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -103,3 +132,40 @@ func TestCompliance(t *testing.T) {
 		})
 	}
 }
+
+// configureDUT configures port1-3 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	p1 := dut.Port(t, "port1")
+	p2 := dut.Port(t, "port2")
+	p3 := dut.Port(t, "port3")
+
+	dut.Config().Interface(p1.Name()).Replace(t, dutPort1.NewInterface(p1.Name()))
+	dut.Config().Interface(p2.Name()).Replace(t, dutPort2.NewInterface(p2.Name()))
+	dut.Config().Interface(p3.Name()).Replace(t, dutPort3.NewInterface(p3.Name()))
+
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(*nonDefaultNI)
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static")
+	dut.Config().NetworkInstance(*nonDefaultNI).Replace(t, ni)
+
+	nip := dut.Config().NetworkInstance(*nonDefaultNI)
+	fptest.LogYgot(t, "nonDefaultNI", nip, nip.Get(t))
+}
+
+// configreATE configures port1-3 on the ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
+	top := ate.Topology().New()
+
+	p1 := ate.Port(t, "port1")
+	p2 := ate.Port(t, "port2")
+	p3 := ate.Port(t, "port3")
```

Comment:
Can you add the topology with the specific port names like port1, port2, port3 for both dut and ate in README or test code?

Reply:
Done